templates/events/show.html.twig

```diff
@@ -166,7 +166,11 @@
             </ul>
             <div class="committee-event-description text--with-bullet b__nudge--bottom-huge l__wrapper--slim">
                 {%- block main_section -%}
-                    {{- event.description -}}
+                    {%- if event.isCitizenAction -%}
```

Comment:
```twig
{{ event.isCitizenAction ? event.description : event.description|raw }}
```

Reply:
Does not work as expected, always act like `{{ event.description }}`